modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'evss/error_middleware'
+
+module ClaimsApi
+  module V2
+    class VeteranIdentifierController < ApplicationController
+      skip_before_action :authenticate, only: %i[show]
+      ICN_FOR_TEST_USER = '1012667145V762142'
+
+      def show
+        headers_to_validate = %w[Authorization X-VA-SSN X-VA-First-Name X-VA-Last-Name X-VA-Birth-Date]
+        validate_headers(headers_to_validate)
```

Comment:
this isnt taking these values through the header. it needs to be a post with a post body of these 4 items.

Reply:
> this isnt taking these values through the header

Hmmm the header values are coming through properly.  Are you saying that the endpoint shouldn't be expecting headers?

Reply:
sorry for the confusion! yep! post is good 👍🏻